lib/client/http.js

```diff
@@ -27,6 +27,7 @@ HttpClient = Client.extend(
             assert(_.isString(options.username), "options.username is expected to be a string");
             assert(_.isString(options.password), "options.password is expected to be a string");
             assert(_.isString(options.database), "options.database is expected to be a string");
+            assert(_.isNumber(options.max_batch), "options.max_batch is expected to be a number");
```

Comment:
Yes, you are right, if we creating client via `createClient()` **without** passing `max_batch`, it is defaulted by [simple extending](https://github.com/gobwas/influent/blob/master/lib/client.js#L9) in abstract `Client` constructor, with value of [here](https://github.com/gobwas/influent/blob/master/lib/client/http.js#L196). So, as you can see, if we pass `config` smth like `{... max_batch: "abc"}` then it will be invalid for use as number. 

So, I think `HttpClient` should not require it strictly, but it could be optional for some purposes like yours below =) And if it passed, it should be a `number`.